app/build.gradle.kts

```diff
@@ -43,7 +44,7 @@ android {
         debug {
             applicationIdSuffix = NiaBuildType.DEBUG.applicationIdSuffix
         }
-        val release by getting {
+        val release = getByName("release") {
```

Comment:
Are these two lines preforming different operations? To me they are identical 🤔

Reply:
One currently works with the Baseline Profile Gradle Plugin, the other doesn't.